deploy.py

```diff
@@ -2,58 +2,71 @@
 import json
 import os
 
-import yaml
-
-from utils import *
+from modules.splunkcloud import SplunkCloudConnector
+from modules.aws_s3 import AwsS3Connector
+from modules.apps_processing import AppFilesProcessor
 
 # FOR LOCAL TESTING
 # from dotenv import load_dotenv
 # load_dotenv(dotenv_path="local.env")
 
+SPLUNK_USERNAME = os.getenv("SPLUNK_USERNAME")
+SPLUNK_PASSWORD = os.getenv("SPLUNK_PASSWORD")
+SPLUNK_TOKEN = os.getenv("SPLUNK_TOKEN")
+
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+
+
 def main():
     if len(sys.argv) != 2:
         print("Usage: python script.py <path_to_yaml_file>")
         sys.exit(1)
 
     yaml_file_path = "environments/" + sys.argv[1] + "/deployment.yml"
 
+    # Initiate deployment report
     deployment_report = {}
 
-    try:
-        data = read_yaml(yaml_file_path)
-    except FileNotFoundError:
-        print(f"Error: The file '{yaml_file_path}' was not found.")
-    except yaml.YAMLError as e:
-        print(f"Error parsing YAML file: {e}")
-        sys.exit(1)
+    # Initiate AppFilesProcessor object
+    app_processor = AppFilesProcessor(yaml_file_path)
```

Comment:
Especially in case of env vars:
```suggestion
    app_processor = AppFilesProcessor()
```